parser-typechecker/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -417,7 +418,7 @@ loop = do
           UpdateBuiltinsI -> "builtins.update"
           MergeBuiltinsI -> "builtins.merge"
           MergeIOBuiltinsI -> "builtins.mergeio"
-          PullRemoteBranchI orepo dest ->
+          PullRemoteBranchI orepo dest _syncMode ->
```

Comment:
Should this key off the `syncMode` to print either `pull` or `debug.pull-exhaustive`?

Reply:
That's a good idea.  My thinking was that this doesn't matter, since `pull` and `debug.pull-exhaustive` have the safe effect, but different side effects.  But maybe showing one or the other would be less surprising though, wdyt?